index.html

```diff
@@ -570,6 +570,25 @@ <h2 id="constrainable-properties">
                 again.</p>
               </td>
             </tr>
+            <tr id="def-constraint-suppressLocalAudioPlayback">
+              <td><dfn>suppressLocalAudioPlayback</dfn></td>
+              <td>{{ConstrainBoolean}}</td>
+              <td>
+                <p>As a constraint, this value is only meaningful if the user
+                selects capturing audio+video from a tab.
+                In that case, a value of <code>true</code> indicates that the
+                user agent SHOULD perform <a>local audio playback
+                suppression</a> on the captured tab.</p>
```

Comment:
Let's keep the name `suppressLocalAudioPlayback` though, since `suppressTargetAudioPlayback` might be misunderstood to mean muting in the capture.

We could still use "target" in implementer prose, e.g. rename the algorithm to <a>target audio playback suppression</a>, but I don't have a strong opinion there either.

Reply:
> I'm unsure about the "local" term, since its opposite is remote, with both having overloaded connotations in most screen-sharing use cases. But I'm also having trouble thinking of better names. What do you think about s/local/target/ ?

I think "local" is the lesser evil, especially as you agree we should keep the name `suppressLocalAudioPlayback`. I don't think it would be reasonable to use a different term in the description and expect readers to understand that no distinction between "local" and "target" is intended. One concept - one word; at least within this section.

> Lastly, I'd like a paragraph stating this MUST NOT be observable by the document(s) in the captured browser display-surface. This should clarify this is a UA mute type of thing — e.g. wouldn't affect a site rendering an audio visualizer.

Is this meant as future-proofing, or is there some existing mechanism for observing this state?